### ISSUE TITLE
Should be mish used before or after batchnorm?

### ISSUE THREAD

**Issue by hadaev8** [digantamisra98/Mish#20]

**Comment by digantamisra98** [comment]
I would suggest Activation (Mish) -> Batch Normalization. However, the original paper and even the official ResNet code in Keras does the opposite. B.N after Mish makes more sense to me from both mathematical and practical sense to me. For reference please follow this [Reddit](https://www.reddit.com/r/MachineLearning/comments/67gonq/d_batch_normalization_before_or_after_relu/) Thread.

**Comment by hadaev8** [comment]
Thanks
